src/mlpack/tests/cne_test.cpp

```diff
@@ -127,7 +139,7 @@ BOOST_AUTO_TEST_CASE(CNELogisticRegressionTest)
     testResponses[i] = 1;
   }
 
-  CNE opt(30, 500, 0.2, 0.2, 0.3, 65, -1);
+  CNE opt(200, 10000, 0.2, 0.2, 0.3, 65, -1);
```

Comment:
hm, maybe we instead of increase the iterations we could train a few times and break once a single run is successful, in favor of the travis test time; not sure that is necessary here.

Reply:
I did some simple test comparison; on my system the old test takes ~0.1s whereas the new one takes more like ~0.3s.  That's short enough that I'm not sure it makes sense to change the strategy here.  What do you think?  I can easily change it if you like.

Reply:
If the test time doesn't blow up, I don't think we have to change anything, let's wait for the travis timings.

Reply:
Okay, about 5 seconds for the CNE test suite, so I agree no need to change the strategy here.